index.html

```diff
@@ -957,8 +957,9 @@ <h2 id="arrays">Array Functions</h2>
         Merges together the values of each of the <b>arrays</b> with the
         values at the corresponding position. Useful when you have separate
         data sources that are coordinated through matching array indexes.
-        If you're working with a matrix of nested arrays, <tt>_.zip.apply</tt>
-        can transpose the matrix in a similar fashion.
+        Use with <tt>apply</tt> to pass in an array of arrays.
```

Comment:
"use with `apply`" isn't really clear here. Can we revert this back to `_.zip.apply`?